.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -4,7 +4,7 @@ contact_links:
     url: https://github.com/home-assistant/home-assistant-polymer/issues
     about: This is the issue tracker for our backed. Please report issues with the UI in the frontend repository.
```

Comment:
This is the issue tracker for our backend.

Reply:
Nice find 👀